R/utilities-pk-analyses.R

```diff
@@ -0,0 +1,26 @@
+#' Calculates the pkAnalyses for all output values available in \code{results}.
+#'
+#' @param results Results of simulation. Typically the \code{results} are calculated using \code{runSimulation} or imported from csv file via \code{importResults}
+#' @param simulation Instance of a simulation for which pk-analyses should be calculated
+#'
+#' @return A list of QuantityPKParameter (one object for each output-pk parameter combination)
+#'
+#' @examples
+#'
+#' simPath <- system.file("extdata", "simple.pkml", package = "ospsuite")
+#' sim <- loadSimulation(simPath)
+#'
+#' addOutputs("Organism|VenousBlood|*|Caffeine", sim)
+#' results <- runSimulation(sim)
+#' pkAnalyses <- getPKAnalyses(results, sim)
+#' @export
+getPKAnalyses <- function(results, simulation) {
+  pkAnalysesTask <- getNetTask("PKAnalysesTask")
+  pkAnalyses <- clrCall(pkAnalysesTask, "CalculateFor", simulation$ref,  as.integer(1),  results$ref)

```

Comment:
This number is the number of indiviudal in the simulation. IT will be either 1 for a single simulation or number of individual in the population. This will be updated  later
Note: We cannot use the length of th results as results only contains entry for sucessful simulations.

Reply:
Please add a TODO-line so we can quickly find the place.

Reply:
I meant this will be updated later when we support population